config/services.yaml

```diff
@@ -944,11 +944,16 @@ services:
         autowire: true
         autoconfigure: true
 
-    # Form type extension
+    # Form type extensions
+    commsy.form.extension.upload_file_type:
+        class: App\Form\Extension\UploadFileTypeExtension
+        tags:
+            - { name: form.type_extension, extended_type: Symfony\Component\Form\Extension\Core\Type\FileType, priority: 1 }
+
     commsy.form.extension.image_type:
         class: App\Form\Extension\ImageTypeExtension
         tags:
-            - { name: form.type_extension, extended_type: Symfony\Component\Form\Extension\Core\Type\FileType }
+            - { name: form.type_extension, extended_type: Symfony\Component\Form\Extension\Core\Type\FileType, priority: 0 }
```

Comment:
Registration of form type extensions is not needed, this is already done by autoconfiguration, see [https://symfony.com/doc/4.4/form/create_form_type_extension.html#registering-your-form-type-extension-as-a-service](https://symfony.com/doc/4.4/form/create_form_type_extension.html#registering-your-form-type-extension-as-a-service)

Reply:
I've added the service configuration explicitly since this would allow to use the `priority` attribute (to specify a load order). But I'll try again without…